.github/workflows/release-notifier.yml

```diff
@@ -0,0 +1,20 @@
+name: Release Contribution Notifier
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release:
+    name: Release Commenter
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
```

Comment:
This may not be needed depending on the repositories `Workflow permissions` settings